e2e-tests/commands/insurance/complete-export-contract-section.js

```diff
@@ -7,6 +7,10 @@
  * @param {Boolean} agentChargeMethodPercentage: Agent charge method is "percentage"
  * @param {Boolean} alternativeCurrency: Should submit an "alternative currency". Defaults to false.
  * @param {Boolean} attemptedPrivateMarketCover: Has attempted to insure through the private market
+ * @param {Boolean} contractAwardedOpenTender: "How was the contract awarded" method as "open tender"
+ * @param {Boolean} contractAwardedNegotiatedContract: "How was the contract awarded" method as "negotiated contract"
+ * @param {Boolean} contractAwardedDirectAward: "How was the contract awarded" method as "direct award"
+ * @param {Boolean} contractAwardedCompetitiveBidding: "How was the contract awarded" method as "competitive bidding"
```

Comment:
Can we use the names of the objects in `EXPORT_CONTRACT_AWARD_METHOD` constants instead of hard coded strings, same as `completeAndSubmitHowWasTheContractAwardedForm`, so that if they change this won't be affected